app/views/RoomsListView/index.js

```diff
@@ -960,7 +981,9 @@ const mapStateToProps = state => ({
 	useRealName: state.settings.UI_Use_Real_Name,
 	appState: state.app.ready && state.app.foreground ? 'foreground' : 'background',
 	StoreLastMessage: state.settings.Store_Last_Message,
-	rooms: state.room.rooms
+	rooms: state.room.rooms,
+	queueSize: state.inquiry.queued.length,
```

Comment:
I'm thinking if we need a selector for this.
WDYT?

Reply:
I agree that is a good change